test/e2e/cypress/e2e/sap_systems_overview.cy.js

```diff
@@ -243,13 +243,11 @@ context('SAP Systems Overview', () => {
   describe('SAP Systems Tagging', () => {
     before(() => {
       cy.removeTagsFromView();
+      cy.navigateToItem('SAP Systems');
+      cy.url().should('include', '/sap_systems');
```

Comment:
I collapsed the siblings `before` statements into one. 

This is enforced bt the `mocha/no-sibling-hooks` rule